acme_diags/derivations/acme.py

```diff
@@ -415,6 +401,26 @@ def cosp_bin_sum(
     return cld_bin_sum
 
 
+def determine_tau(tau, tau_low0, tau_high0):
```

Comment:
I would add unit tests to this new function, similar to the ones here: https://github.com/E3SM-Project/e3sm_diags/blob/master/tests/acme_diags/derivations/test_acme.py